aether-ui/aether/ui/assets/package.json

```diff
@@ -29,7 +29,7 @@
     "react": "~16.10.0",
     "react-dom": "~16.10.0",
     "react-onclickoutside":"~6.9.0",
-    "react-intl": "~2.9.0",
+    "react-intl": "~3.3.2",
```

Comment:
Remember, you need to replace the `enzyme-react-intl` package to make the tests pass.